modules/rbac-system-roles-3630.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control.adoc
+:_module-type: CONCEPT
+[id="rbac-system-roles-3630_{context}"]
+= System roles
+
+[role="_abstract"]
+{product-title} includes some default system roles that you can apply on users.
```

Comment:
"apply on users" sounds odd, maybe "apply to users"?

Reply:
Updated it to `apply to users when you create rules`.